app/dturn/__tests__/rank-statements.js

```diff
@@ -24,37 +16,38 @@ const OPTIONS = {
 describe('Test ranking scenarios', () => {
   beforeAll(async () => {
     await initDiscussion(DISCUSSION_ID, OPTIONS)
-    const props = []
-    for (let i = 0; i < 20; i++) {
-      props.push([DISCUSSION_ID, `user${i}`, `statement${i}`])
+
+    const totalStatements = OPTIONS.group_size * 2 - 1
+    const insertPromises = []
+    for (let i = 0; i < totalStatements; i++) {
+      insertPromises.push(insertStatementId(DISCUSSION_ID, `user${i}`, `statement${i}`))
     }
-    for await (const args of props) {
-      await insertStatementId(...args)
+    await Promise.all(insertPromises)
+
+    console.log(`Checking if ${USER_ID} exists in discussion...`)
+    if (!Discussions[DISCUSSION_ID].Uitems[USER_ID]) {
+      console.log(`Inserting statement for ${USER_ID} to ensure it joins the discussion`)
+      await insertStatementId(DISCUSSION_ID, USER_ID, `statement-${USER_ID}`)
     }
   })
 
   test('Can rank 2 statements as most important', async () => {
     const statements = await getStatementIds(DISCUSSION_ID, 0, USER_ID)
-
-    // Ensure statements is defined before proceeding
     expect(statements).toBeDefined()
     expect(statements.length).toBe(10)
 
+    console.log('Retrieved statements:', statements)
```

Comment:
okay, now that this code is working isn't it time to take the console.log statements out.